Unity-Technologies/com.unity.probuilder
# Feature pathfinding faces

### Purpose of this PR

Added a path-finding feature to select faces in the shortest path between 2 given faces

### Comments to Reviewers

Another pull request will also add new tests on the caching and a feature to preview the path before clicking.
How to test: 
1. Select a face
2. Hold ctrl+shift or cmd+shift
3. Select a second face

A path between those 2 faces should be selected
